test/integration/outreach/outreach.js

```diff
@@ -42,6 +43,7 @@ context('Outreach', function() {
     cy
       .get('.js-submit')
       .click()
+      .wait('@routeFormAction')
```

Comment:
This was just a missing wait.. Ideally tests shouldn't need to chang for this PR